pyca/capture.py

```diff
@@ -92,9 +92,12 @@ def safe_start_capture(event):
         logger.error('Recording failed')
         logger.error(traceback.format_exc())
         # Update state
-        recording_state(event.uid, 'capture_error')
-        update_event_status(event, Status.FAILED_RECORDING)
-        set_service_status_immediate(Service.CAPTURE, ServiceStatus.IDLE)
+        try:
+            set_service_status_immediate(Service.CAPTURE, ServiceStatus.IDLE)
+            recording_state(event.uid, 'capture_error')
+            update_event_status(event, Status.FAILED_RECORDING)
+        except Exception:
+            logger.error('Could not update recording status')
```

Comment:
I think this error should also be logged.

```suggestion
            logger.error('Could not update recording status')
            logger.error(traceback.format_exc())
```

Reply:
> Also I am curios. Why is it `logger.error(traceback.format_exc())`

Because either this did not exist back when I wrote this or I did not know better.

Looking at the [logging statement documentation](https://docs.python.org/3/library/logging.html#logging.Logger.debug) it might also make sense to use `exc_info` or `stack_info` within the logger statements